x-pack/test/apm_api_integration/tests/alerts/rule_registry.ts

```diff
@@ -120,20 +133,16 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
   registry.when('Rule registry with write enabled', { config: 'rules', archives: [] }, () => {
     it('does not bootstrap indices on plugin startup', async () => {
-      const { body } = await es.indices.get({
-        index: ALERTS_INDEX_TARGET,
-        expand_wildcards: 'open',
-        allow_no_indices: true,
-      });
-
-      const indices = Object.entries(body).map(([indexName, index]) => {
-        return {
-          indexName,
-          index,
-        };
-      });
-
-      expect(indices.length).to.be(0);
+      const { body: targetIndices } = await getAlertsTargetIndices();
+      try {
+        await es.indices.get({
+          index: targetIndices[0],
+          expand_wildcards: 'open',
+          allow_no_indices: true,
+        });
+      } catch (exc) {
+        expect(exc.statusCode).to.eql(404);
```

Comment:
there's no assertion here when the request succeeds, no?

Reply:
Good catch I'll update this to fail if a 200 response is received.

Reply:
Fixed here: 6c5e2bfaeb4bf6016ed016e37eb2c7cea7c6fce7